package.json

```diff
@@ -32,6 +32,7 @@
     "@fortawesome/free-brands-svg-icons": "^5.12.0",
     "@fortawesome/free-solid-svg-icons": "^5.11.2",
     "@fortawesome/react-fontawesome": "^0.1.7",
+    "@storybook/addon-actions": "^6.0.28",
```

Comment:
I am surprised this even works!
Please run `npm install @storybook/addon-actions@5.3.17` since we shouldn't mix different versions of storybook packages.